{"uuid": "0e0d21ff-279b-4f03-883a-3b2a608b22e0", "children": ["60640a9c-7252-456c-ba19-6d392e573362"], "befores": [{"name": "take_screenshot_when_failure", "status": "passed", "start": 1559914946911, "stop": 1559914946911}], "afters": [{"name": "take_screenshot_when_failure::1", "status": "passed", "start": 1559914952205, "stop": 1559914952206}, {"name": "take_screenshot_when_failure::tear_down", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/allure_commons/_allure.py\", line 169, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/ace/Documents/GitHub/epam_ta/pom/conftest.py\", line 25, in tear_down\n    if request.node.rep_call.failed:\n"}, "start": 1559914952206, "stop": 1559914952206}], "start": 1559914946911, "stop": 1559914952206}